README.md

```diff
@@ -109,3 +109,8 @@ $ docker logs --tail 100 postgres-upgrade-testing
 $ # can now safely remove "$OLD"
 $ sudo rm -rf "$OLD"
 ```
+## Upgrading with extensions:
+If your current postgres installation has extensions installed, for example [TimescaleDB](https://www.timescale.com/), you'll have to create a custom container that has the needed extensions installed in both the old and new version of Postgres.
+Additionally, supplying an option to pre-load any required libraries in the upgrade process is needed (`"-c shared_preload_libraries=YOUR_PLUGIN"`).
+
+For a simple example with TimescaleDB, refer to [this comment](https://github.com/timescale/timescaledb/issues/2260#issuecomment-686786971).
```

Comment:
Thanks!  Just some minor formatting adjustments I'd like to see: :eyes:

````suggestion
```

## Upgrading with PostgreSQL extensions

If your current installation has extensions installed (for example, [TimescaleDB](https://www.timescale.com/)), you'll have to create a custom image that has the needed extensions installed in *both* the old and new versions of PostgreSQL.  Additionally, it is necessary to supply an option to pre-load any required libraries in the upgrade process (for example, `-c shared_preload_libraries=YOUR_PLUGIN`).

For a simple example with TimescaleDB, refer to [timescale/timescaledb#2260](https://github.com/timescale/timescaledb/issues/2260#issuecomment-686786971).
````

Reply:
Looks good 👍